src/auth/app/service.go

```diff
@@ -27,7 +27,10 @@ func RegisterService(
 
 	var authenticatorService *authenticator.Authenticator
 	if jwtValidator != nil || apikeyClaims != nil || rootCAs != nil {
+		logger.Info("authentication middleware is enabled")
 		authenticatorService = authenticator.New(jwtValidator, apikeyClaims, rootCAs, logger)
+	} else {
```

Comment:
an `if` statement only to log is an architecture anti-pattern and should normally never happen

Reply:
I could redefine it as:
```
if authenticatorService == nil 
```

or just not log in this case, but IMO it is important Warn clients auth is not enabled